server/src/service.rs

```diff
@@ -273,6 +273,13 @@ impl Service {
                 Arc::new(Keyring::open("login", secret).await?),
             );
             collections.push(collection.clone());
+            {
```

Comment:
This commit has no use cases so it should be dropped and added with whatever patch that would make use of it